etsin_finder/frontend/js/index.css

```diff
@@ -0,0 +1,16 @@
+@font-face {
+  font-family: Lato;
+  src: local('Lato') url(../resources/fonts/lato/Lato-Regular.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Lato;
+  src: local('Lato') url(../resources/fonts/lato/Lato-Bold.ttf);
+  font-weight: 700;
+}
+
+@font-face {
+  font-family: Lato;
+  src: local('Lato') url(../resources/fonts/lato/Lato-Thin.ttf);
+  font-weight: 300;
+}
```

Comment:
The src values need to be separated with commas, and the local names need to be more specific. This is what worked for me:
```
@font-face {
  font-family: Lato;
  src: local('Lato Regular'), url(../resources/fonts/lato/Lato-Regular.ttf) format('truetype');
}
@font-face {
  font-family: Lato;
  src: local('Lato Bold'), url(../resources/fonts/lato/Lato-Bold.ttf);
  font-weight: 700;
}
@font-face {
  font-family: Lato;
  src: local('Lato Thin'), url(../resources/fonts/lato/Lato-Thin.ttf);
  font-weight: 300;
}
```